wp-content/themes/borderzine/less/widgets.less

```diff
@@ -10,4 +10,28 @@
             margin-right: 10px;
         }
     }
+}
+
+.borderzine-6-col-widget {
+    .widgettitle {
+        display: inline-block;
+    }
+    p.morelink {
+        margin-left: 10px;
+        a {
+            color: @white;
+        }
+    }
+    ul {
+        display: flex;
+        justify-content: space-between;
+        flex-wrap: wrap;
+        li {
+            max-width: 175px;
+            img {
+                width: 100%;
+                height: 150px;
```

Comment:
This is causing image aspect ratio distortion with the featured images. 

![Screen Shot 2019-10-07 at 19 52 30 ](https://user-images.githubusercontent.com/1754187/66357305-449d9b80-e93c-11e9-96d7-055b1f33eb62.png)

I'm not sure if we can expect Borderzine to use featured images of a fixed aspect ratio, so perhaps instead of giving people the choice or hardcoding to `large` or `medium` as the image size here, could we use `rect_thumb_half` from Largo's defined image sizes? It should be 400x300, cropped.

https://github.com/INN/largo/blob/f50df16e9e06fb9402802e2f1ce76ce058d26d76/functions.php#L268

Reply:
@benlk https://github.com/INN/umbrella-borderzine/pull/75/commits/f766833d9593870549f624102f4c410abd663258 updates the css to use `display:grid;` instead of flex and `grid-template-columns: repeat(6, 1fr)` so they're in rows of 6.